main.cpp

```diff
@@ -2,7 +2,56 @@
 
 using namespace std;
 
+void menuShow();
+void menuChoose();
+
 int main() {
-	cout << "Hello fellow Students!" << endl;
+	menuShow();
+    menuChoose();
 	return 0;
-}
\ No newline at end of file
+}
+
+void menuShow() {
+	system("cls");
+	cout << "Program option: " << endl;
+	cout << "1. Show students list " << endl;
+	cout << "2. Add new student " << endl;
+	cout << "3. Remove student " << endl;
+	cout << "4. Sort students list " << endl;
+	cout << "5. Exit program " << endl;
+}
+
+void menuChoose() {
+    char option;
+
+    do { 
+        cin >> option;
+
+        switch (option) {
+            case '1' : {
+                           cout << "Showing students list " << endl;
```

Comment:
To samo co w przypadku wyswietlania menu, zbedne spacje na koncu linii
Sorry ze nie zauwazylem wczesniej, ale skladnia instrukcji switch nie wymaga nawiasow klamrowych po case, usun je prosze.